ocrd_calamari/recognize.py

```diff
@@ -31,6 +31,14 @@ def _init_calamari(self):
         checkpoints = glob(self.parameter['checkpoint'])
         self.predictor = MultiPredictor(checkpoints=checkpoints)
 
+        self.input_channels = self.predictor.predictors[0].network.input_channels
+        #self.input_channels = self.predictor.predictors[0].network_params.channels # not used!
+        # binarization = self.predictor.predictors[0].model_params.data_preprocessor.binarization # not used!
+        # self.features = ('' if self.input_channels != 1 else
+        #                  'binarized' if binarization != 'GRAY' else
+        #                  'grayscale_normalized')
```

Comment:
Is this commented-out code necessary?

Reply:
Yes, I think it is. It was hard enough arriving at this with the poor documentation in Calamari itself – only to see that it does not actually work with the models we currently get, because they don't have binarization configured in. So the comment could be revived later-on, or at least shows what would be necessary to do.